builder/config.conf

```diff
@@ -0,0 +1,8 @@
+export QT_BASE="/opt/Qt/5.14.1"
+export PYTHON_BASE="/Library/Frameworks/Python.framework/Versions/3.7"
```

Comment:
is it ok to use system Python rather than homebrew's?

Reply:
I haven't come to use python yet, will see later in the process